package.json

```diff
@@ -18,31 +18,33 @@
     "url": "https://github.com/pixijs/pixi.js.git"
   },
   "scripts": {
-    "clean": "rimraf bin && mkdirp bin",
+    "clean": "rimraf dist lib && mkdirp dist && mkdir lib",
     "prestart": "npm run clean",
     "start": "parallelshell \"npm run watch:lint\" \"npm run watch\"",
-    "watch": "pixify -n PIXI -o pixi -w",
+    "watch": "npm run build -- --watch",
     "watch:lint": "watch \"eslint scripts src test || exit 0\" src",
+    "watch:bundle": "npm run bundle -- -w",
     "test": "floss --path test/index.js",
     "test:debug": "npm test -- --debug",
     "prerenders": "npm --prefix scripts/renders i scripts/renders",
     "renders": "electron scripts/renders",
-    "precoverage": "rimraf coverage && npm run build -- --noExternal",
-    "coverage": "npm test -- -c bin/pixi.js -s -h",
+    "precoverage": "rimraf coverage && npm run build",
+    "coverage": "npm test -- -c dist/pixi.js -s -h",
     "lint": "eslint scripts src test --max-warnings 0",
     "lintfix": "npm run lint --fix",
-    "prebuild": "npm run lint && npm run clean",
-    "build": "pixify -n PIXI -o pixi",
+    "prebuild": "npm run lint",
+    "build": "babel src --out-dir lib -s",
+    "bundle": "pixify -d dist -n PIXI -o pixi",
```

Comment:
if you use gulp source maps you can tell it to "load sourcemaps" so that it honors what is in the file. Don't think you can do that same using browserify itself though